opencga-master/src/main/java/org/opencb/opencga/master/monitor/daemons/AuthorizationDaemon.java

```diff
@@ -44,81 +32,83 @@ public class AuthorizationDaemon extends MonitorParentDaemon {
 
     public AuthorizationDaemon(int interval, String sessionId, CatalogManager catalogManager) throws CatalogDBException {
         super(interval, sessionId, catalogManager);
-        this.studyDBAdaptor = dbAdaptorFactory.getCatalogStudyDBAdaptor();
+//        this.studyDBAdaptor = dbAdaptorFactory.getCatalogStudyDBAdaptor();
         this.authorizationManager = catalogManager.getAuthorizationManager();
     }
 
     @Override
     public void run() {
-
-        Query allStudies = new Query();
-        QueryOptions options = new QueryOptions(QueryOptions.INCLUDE,
-                        Arrays.asList(StudyDBAdaptor.QueryParams.PERMISSION_RULES.key(), StudyDBAdaptor.QueryParams.ID.key(),
-                                StudyDBAdaptor.QueryParams.UID.key()));
-
-        while (!exit) {
-            try {
-                Thread.sleep(interval);
-            } catch (InterruptedException e) {
-                if (!exit) {
-                    e.printStackTrace();
-                }
-            }
-            logger.info("----- AUTHORIZATION DAEMON -----", TimeUtils.getTimeMillis());
-
-            try (DBIterator<Study> iterator = studyDBAdaptor.iterator(allStudies, options)) {
-                while (iterator.hasNext()) {
-                    applyPermissionRules(iterator.next());
-                }
-            } catch (Exception e) {
-                logger.error("{}", e.getMessage(), e);
-            }
-        }
+//
```

Comment:
Are we sure of removing this?

Reply:
For some reason, that class was not in use and it wasn't compatible with the Organization changes. If needed, we can fix it to make it compatible with Organizations and recover the functionality.